src/mscorlib/src/System/Diagnostics/Contracts/ContractsBCL.cs

```diff
@@ -202,11 +202,6 @@ public void SetUnwind()
         }
     }
 
-#if FEATURE_SERIALIZATION
-    [Serializable]
-#else
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1032:ImplementStandardExceptionConstructors")]
-#endif
     [SuppressMessage("Microsoft.Design", "CA1064:ExceptionsShouldBePublic")]
```

Comment:
Is the removal of `[Serializable]` on `ContractException` intentional? I think it should stay.